src/System.Net.WebHeaderCollection/tests/project.lock.json

```diff
@@ -3,28 +3,15 @@
   "version": -9996,
   "targets": {
     "DNXCore,Version=v5.0": {
-      "System.Collections/4.0.0-beta-22809": {
+      "System.Collections/4.0.10-beta-23024": {
         "dependencies": {
-          "System.Runtime": "4.0.0-beta-22809"
-        },
-        "compile": {
-          "ref/any/System.Collections.dll": {}
-        }
-      },
-      "System.Collections.NonGeneric/4.0.0-beta-23024": {
-        "dependencies": {
-          "System.Runtime": "4.0.20-beta-23024",
-          "System.Globalization": "4.0.10-beta-23024",
-          "System.Resources.ResourceManager": "4.0.0-beta-23024",
-          "System.Diagnostics.Debug": "4.0.10-beta-23024",
-          "System.Threading": "4.0.10-beta-23024",
-          "System.Runtime.Extensions": "4.0.10-beta-23024"
+          "System.Runtime": "4.0.20-beta-23024"
         },
         "compile": {
-          "ref/dotnet/System.Collections.NonGeneric.dll": {}
+          "ref/dotnet/System.Collections.dll": {}
```

Comment:
System.Net.WebHeaderCollection is a new contract for both CoreFx and .NETNative.  I don't think it is a problem to change the *implementation* dependency.  We're not changing the *contract* dependency.  In terms of NuGet packaging, etc., I think this is an ok change.

@ericstj Can you confirm this?